### ISSUE TITLE
Compile Time Error w/GM Long Term Stable (LTS) build 

### ISSUE THREAD

**Issue by GentooGames** [nommiin/ImGui_GM#4]
build: Long Term Stable
runtime version: v2022.0.0.12

error: Script: ImGui at line 4092 : Assignment operator expected

![image_2023-02-27_175837603](https://user-images.githubusercontent.com/22380735/221724560-3f73a580-6df9-4df8-8beb-8214088d0636.png)

**Comment by nommiin** [comment]
Hello! So, this is a bit of a tricky one since I hadn't really tested ImGui_GM with LTS at all... plus the fact that the extension sorta hinges on the changes to the `static` keyword made since LTS. 

Regarding your error above, seems like LTS doesn't have support the new numeric separators either. I'll look into LTS compatibility for ImGui, worse comes to worse you might be able to get away with using [`snake_case.gml`](https://github.com/nommiin/ImGui_GM/blob/main/snake_case.gml) from this repo on LTS

**Comment by nommiin** [comment]
Hi again! So, I've released [v1.0.11](https://github.com/nommiin/ImGui_GM) which gets rid of some of the newer features ImGui_GM used (numeric separators, static_get, etc) and adds in a new compatibility mode that checks if the main script was renamed to "ImGui_" (and creates ImGui as a globalvar if so)

After testing with LTS v2022.0.0.12, it seems like the extension works properly now! I've also made a [Wiki page](https://github.com/nommiin/ImGui_GM/wiki/LTS-Support) with more info pertaining to LTS support.

Let me know if this fixes your issues above!

**Comment by GentooGames** [comment]
Thanks so much for jumping on this quickly! I sent some love (money) over to you on itch lol, haha seriously, thanks again!  🙏 so far everything is working perfectly- gentooOn Mar 3, 2023, at 13:59, Torin ***@***.***> wrote:﻿
Hi again! So, I've released v1.0.11 which gets rid of some of the newer features ImGui_GM used (numeric separators, static_get, etc) and adds in a new compatibility mode that checks if the main script was renamed to "ImGui_" (and creates ImGui as a globalvar if so)
After testing with LTS v2022.0.0.12, it seems like the extension works properly now! I've also made a Wiki page with more info pertaining to LTS support.
Let me know if this fixes your issues above!
